ChangeNotifier isn't O(N2) anymore, as can be seen on the docs. (#626)
rrousselGit/provider: README.md

}
```

> This is not allowed, because the modification has to be immediate.

Reviewer: ```suggestion
This is not allowed, because the state update is synchronous.
```
